client/state/comments/test/selectors.js

```diff
@@ -2,7 +2,7 @@
 /**
  * External dependencies
  */
-import { expect } from 'chai';
+import { expect as chaiExpect } from 'chai';
```

Comment:
This is one way of doing it. Another one is to run jest-codemod. It will update all basic chai assertions for you

Reply:
oh cool!

Reply:
where is that? how do I run it?

Reply:
found it https://facebook.github.io/jest/docs/en/migration-guide.html

Reply:
I'm trying to convince @samuri and others to run prettier on all test files, then we can run jest-codemods on all test files. Only enzyme and sinon specific assertion would have to use chaiExpect workaround 😃